src/middlewared/middlewared/plugins/jail.py

```diff
@@ -949,7 +949,7 @@ def fstab(self, jail, options):
             )
 
             for i in _list:
-                fstab_entry = i[1].split()
+                fstab_entry = i[1]
```

Comment:
`fstab_entry` changes type from `list` to `str`???

Reply:
So with latest `iocage` changes `i[1]` is a list, and in older version, `fstab_entry` is changing `str` to list